src/tests/profiler/native/rejitprofiler/ilrewriter.cpp

```diff
@@ -2,6 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 #include <cstring>
+#include <assert.h>
+
+// Workaround to not link PAL to this test.
+#define _ASSERTE assert
```

Comment:
Instead of this workaround, it may be better to switch `corhlpr.cpp` to use the standard `assert` instead `_ASSERTE`.

Reply:
There also `_ASSERTE` in the 
https://github.com/dotnet/runtime/blob/e2819b239e3eb7aa1062b58b7a27276e8919d630/src/coreclr/inc/corhlpr.h#L262 
should I changes there? This header is not so rarely used as `corhlpr.cpp`.

Reply:
Yes, it is fine to change corhlpr.h to assert too. In general, I think it is a good idea to stick to the standard constructs in anything that is used across boundaries.

Reply:
I meant any code is used e.g. in both coreclr and tests; or in both coreclr and mono.

The `inc` headers are grab-bag of many different parts. I would not generalize this to everything in `inc` headers.

Reply:
Thank you for explanation. It was not clear for me what exactly do you mean, so I try to re-phrase your words how I understand them. Now it make sense.

Reply:
NP. It is always better to ask questions when something is not clear.